nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/test/java/org/nuxeo/ecm/platform/tag/AbstractTestTagService.java

```diff
@@ -99,6 +99,9 @@
     @Inject
     protected HotDeployer deployer;
 
+    @Inject
+    protected NXAuditEventsService nxAuditEventsService;
```

Comment:
Name it `auditEventsService` for consistency with other places where you do that.